demo/src/main/java/com/example/demo/contoller/StudentController.java

```diff
@@ -0,0 +1,99 @@
+
+package com.example.demo.contoller;
+
+import com.example.demo.dto.ResponseDTO;
+import com.example.demo.dto.StudentDTO;
+import com.example.demo.entity.StudentEntity;
+import com.example.demo.impl.IStudentDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/student")
+public class StudentController {
+	   
+	   private static String errorMessage = "No entity with id '{0}' found";
+	   
+	   @Autowired
+	   private IStudentDAO studentDAO;
+	   
+	   @PostMapping(value = "/1.0/save")
+	   @ResponseBody
+	   public ResponseEntity<ResponseDTO> save(@RequestBody StudentDTO studentDTO) {
+			 ResponseDTO responseDTO;
+			 HttpStatus status;
+			 try {
+				    studentDAO.saveStudentInfo(studentDTO);
```

Comment:
Isn't it have a return type? if yes, why ?

Reply:
In StudentDAO::saveStudentInfo method not have any try catch block if any error occurs then it can be handle inside controller method, so DAO method not have any return type